Assets/Scripts/PlayerIcon.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.UI;
+
+/// <summary>
+/// Manages Player Icon Location
+/// Appears when camera switch to map mode and when in isometric view
+/// 
+/// </summary>
+public class PlayerIcon : MonoBehaviour
+{
+    public Transform player;
+    private static bool _isoActive;
+    private static bool _mapActive;
+
+    private void Start()
+    {
+        
+        CameraSwitch.OnMapSwitch += SetMapActive;
```

Comment:
Both events can be subscribed to with the same method.

Reply:
The two events are different. They have different properties to set for and will be called at different times. They cannot be subscribed to the same method.

Reply:
They'll be called at different times, but they do the same thing: they show the player icon.